composer.json

```diff
@@ -212,7 +212,7 @@
         },
         "ps_apiresources": {
             "type": "vcs",
-            "url": "https://github.com/PrestaShop/ps_apiresources"
+            "url": "https://github.com/jolelievre/ps_apiresources"
```

Comment:
@jolelievre Oh wait !

Reply:
yeah it's on purpose and temporary, thats how we handle modifications that impact both the core and the module